tests/virt_autotest/virt_utils.pm

```diff
@@ -34,8 +34,8 @@ sub enable_debug_logging {
     # turn on debug for libvirtd
     my $libvirtd_conf_file = "/etc/libvirt/libvirtd.conf";
     if (!script_run "ls $libvirtd_conf_file") {
-        script_run "sed -i '/log_level *=/{h;s/^[# ]*log_level *= *[0-9].*\$/log_level = 1/};\${x;/^\$/{s//log_level = 1/;H};x}' $libvirtd_conf_file";
-        script_run "sed -i '/log_outputs *=/{h;s%^[# ]*log_outputs *=.*[0-9].*\$%log_outputs=\"1:file:/var/log/libvirt/libvirtd.log\"%};\${x;/^\$/{s%%log_outputs=\"1:file:/var/log/libvirt/libvirtd.log\"%;H};x}' $libvirtd_conf_file";
+        script_run "sed -i '/log_level *=/{h;s/^[# ]*log_level *= *[0-9].*\$/log_level = 3/};\${x;/^\$/{s//log_level = 3/;H};x}' $libvirtd_conf_file";
```

Comment:
After basic analyze, we found there was created most 10Gb libvirtd log on sles15sp3 xen host with log_level setting as 1 
Please add the comment here, why changed log_level from 1 to 3 on sles15sp3 xen host without kvm role firstly. 
Then, just only code changed with log_level as 3 for sles15sp3 xen.

Reply:
Others, LGTM.

Reply:
LGTM. Better if tells warning=3 debug=1

Reply:
All done. 

verification run is in progress: https://openqa.nue.suse.com/tests/4706205 & https://openqa.nue.suse.com/tests/4706207